pkg/sdk/poc/README.md

```diff
@@ -35,17 +35,71 @@ make clean-generator-poc run-generator-poc
 
 ### Next steps
 ##### Essentials
-- use DSL to build object definitions (from branch [go-builder-dsl](https://github.com/Snowflake-Labs/terraform-provider-snowflake/tree/go-builder-dsl)) - ideally leave two options of defining objects and proceed with generation based on definition provided
-- differentiate between different actions implementations (now only `Create` and `Alter` has been considered, `Show` on the other hand has totally different implementation)
-- generate `struct`s for `Show` and `ShowID`
+- modify makefile scripts (see TODO in makefile)
+- add arguments to the generator, so we'll be able to specify which files should be generated / re-generated,
```

Comment:
I would argue if this is the top priority (and this is second on the essentials list)